booking-up-for-beauty/hbhati/booking_up_for_beauty.go

```diff
@@ -0,0 +1,46 @@
+package booking
+
+import (
+	"fmt"
+	"time"
+)
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	// Input: 7/25/2019 13:45:00
+	layout := "1/2/2006 15:04:05"
+	t, _ := time.Parse(layout, date)
```

Comment:
You probably need to handle errors coming from `time.Parse`, probably a shared function to parse given a layout and a date that handles errors could be useful and will lead to better maintainability